js-exercises/curry/curry.js

```diff
@@ -0,0 +1,14 @@
+function curry(func) {
+  return function curried(...args) {
+    if (args.length >= func.length) {
+      return func.apply(this, args);
+    }
+    return function check(...args2) {
```

Comment:
@patelronakp Avoid using random function names. 
If you are not able to come up with a meaningful name, go with the anonymous function.